src/operator/subgraph/mkldnn/mkldnn_fc_sum_fuse.h

```diff
@@ -163,13 +186,21 @@ class SgMKLDNNFCSumFuseProperty : public SubgraphProperty {
     if (ew_add_node != nullptr) {
       CHECK_NOTNULL(fc_node->attrs.subgraphs[0]);
       auto fc_orginal = fc_node->attrs.subgraphs[0]->outputs[0].node;
+
       if (fc_orginal->op() == Op::Get("FullyConnected")) {
         nnvm::Symbol new_sym;
-        nnvm::NodeEntry& ew_input_with_fc = (ew_add_node->inputs[1].node == fc_node)
-                                                ? ew_add_node->inputs[1]
-                                                : ew_add_node->inputs[0];
-        ew_input_with_fc.node             = fc_orginal;
-        new_sym.outputs.emplace_back(ew_add_node);
+        nnvm::ObjectPtr n = nnvm::Node::Create();
```

Comment:
Consider adding comment why elemwise_add node must be placed in subgraph (InferShape/InferType)

Reply:
Done